### ISSUE TITLE
Disable "Launch as admin" combobox for apps that couldn't be launched elevated

### ISSUE THREAD

**Issue by SeraphimaZykova** [JaneaSystems/PowerToys-DevProjects#74]
Adding this issue for visibility. 
Some apps, such as Settings, can't be launched as admin. To avoid launching errors, it'd be better to disable the combo box.

![image](https://github.com/user-attachments/assets/3a8d75de-701c-4a4d-b788-76867966674b)

**Comment by SeraphimaZykova** [comment]
@plante-msft, does it sound good?

**Comment by plante-msft** [comment]
yes this sounds good - any confusion from users as to why it's greyed out can be covered in documentation. Thanks!
